clang/test/dpct/cmake_migration/case_039/expected.txt

```diff
@@ -28,7 +28,7 @@ add_link_options()
 add_link_options()
 add_link_options()
 add_link_options("")
-add_link_options("--no-ptxas-options")
+add_link_options("")
```

Comment:
Fix bus issue  rule [rule_no_ptxas_options_add_link_options](https://github.com/oneapi-src/SYCLomatic/blob/1101154f460f01ec96a97a7c8f74da91eba0e11d/clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml#L2083) does not work.